pinc/filter_project_list.inc

```diff
@@ -157,8 +158,7 @@ class DifficultyElement extends ProjectFilterElement
             echo " checked";
         }
         echo ">\n";
-        $difficulty_options = ['beginner' => _("Beginners Only"), 'easy' => _("Easy"), 'average' => _("Average"), 'hard' => _("Hard")];
-        foreach ($difficulty_options as $value => $option) {
+        foreach (get_project_difficulties() as $value => $option) {
             echo "<label for='$value'>&nbsp;$option</label><input type='checkbox' name='difficulty[]' class='diff-opt' id='$value' value='$value'";
```

Comment:
ideally we'd encode in case french or something introduced some ' characters.

Reply:
Good catch! I've escaped `$option`.